features/step_definitions/scheduler.rb

```diff
@@ -21,7 +21,14 @@
   patch_json = org_scheduler.to_json
   _admin = admin
   teardown_add {
-    opts = {resource: 'scheduler', resource_name: name, p: patch_json, type: 'merge' }
+    # Added code to support removal of tlsSecurityProfile while restoring
+    @result = admin.cli_exec(:get, resource: crd, resource_name: name, o: 'yaml')
+    if @result[:success] and @result[:parsed]['spec']['tlsSecurityProfile']
+      patch_json = [{"op": "remove","path": "/spec/tlsSecurityProfile"}].to_json
+      opts = {resource: crd, resource_name: name, p: patch_json, type: 'json' }
```

Comment:
We only update CRD in this case, or we want to update CRD first, then scheduler later ?

Reply:
@liangxia the existing function did not work to update the CRD when the spec contains 'tlsSecurityProfile'. so modified the func to get that working. In this case we only update the CRD.

Reply:
Ack